### DISCUSSION TITLE
PostgreSQL numeric values rounded up to 3 fractional digits

### DISCUSSION THREAD

**Discussion by ced75** [dbeaver/dbeaver#13248] [Q&A]
#### System information: 
- Operating system (distribution) and version : Linux Ubuntu 18.04
- DBeaver version : 21.1.2

#### Connection specification:
- Database name and version : PostgreSQL 12
- Driver name : postgresql jdbc driver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No proxy, no SSH

#### Describe the problem you're observing:
Display of NUMERIC fields is limited to 3 digits, even when data formats are configured to 10 maximum fraction digits.
In the database, the fields are defined as NUMERIC, with no scale and no precision.

#### Steps to reproduce, if exist:
Here are my Preferences for numbers display:

- Use Grouping : unticked (but ticked gives the same result)
- Maximum fraction digits: 10 (but increased value gives the same result)
- Minimum fraction digits: 0
- Use data type scale for fraction digits: unticked (but ticked gives the same result)
- Rounding mode : UNNECESSARY (but empty gives the same result)

These preferences are applied to all connections.

Example of SQL code to reproduce the problem:
```sql
CREATE TEMPORARY TABLE toto (id INT4, val1 NUMERIC, val2 NUMERIC);

INSERT INTO toto
VALUES (1, 125.1578649, 1335.879654)
, (2, 256.178654, 157.986);

SELECT *
FROM toto;
```

The result looks like this:
![image](https://user-images.githubusercontent.com/12892581/124736239-184eee80-df17-11eb-9a58-dded14482fb2.png)

And I would get all the digits for all values.

Thanks in advance for your help.
Kind regards,

Cedric

**Comment by uslss** [comment]
unfortunately, cannot reproduce it. 
![image](https://user-images.githubusercontent.com/31996417/124749383-d335b700-df2c-11eb-90ba-877ce3857854.png)

**Comment by ced75** [comment]
So, it could be related to the language and country parameters.
Mine are fr - français for language and FR - France for country.

**Comment by ced75** [comment]
If I disable numeric formatting, the result is correctly displayed.
